flink-libraries/flink-table/src/main/scala/org/apache/flink/table/codegen/CodeGenerator.scala

```diff
@@ -255,17 +256,23 @@ class CodeGenerator(
     *
     * @return A GeneratedAggregationsFunction
     */
-  def generateAggregations(
+  def generateAggregations[T](
```

Comment:
revert all changes in `CodeGenerator`. The wrapping functions (such as `MapRunner`) can extract the `Row` from a `CRow` and call the generated function with that object.